resource/js/app.js

```diff
@@ -21,10 +22,13 @@ crowi.fetchUsers();
 const crowiRenderer = new CrowiRenderer();
 window.crowiRenderer = crowiRenderer;
 
+const pageId = $('#content-main').data('page-id');
```

Comment:
We can use `document.querySelector('#content-main')`
see: https://github.com/crowi/crowi/pull/174/files#diff-39992ee08af86834f2b5095ded60c098R25